samples/graph-app-installation-lifecycle/csharp/AppInstallation/Manifest/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Where are we using images present in this folder?  https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/v-trkum/appInstallation/samples/graph-app-installation-lifecycle/csharp/AppInstallation/Images

Reply:
done